modules/juce_opengl/opengl/juce_OpenGLGraphicsContext.cpp

```diff
@@ -130,6 +151,7 @@ struct CachedImageList  : public ReferenceCountedObject,
         CachedImageList& owner;
         ImagePixelData* pixelData;
         OpenGLTexture texture;
+        const void* lastDisplay = nullptr;

```

Comment:
Surround with `#if JUCE_MAC` ?

Reply:
Is a better name for this perhaps `curDisplay`? Unlike the other `lastDisplay`, you're never comparing cur to last here (this becomes more convincing if you decided to, say, call `lastDisplay` `prevDisplay` instead).